docs/guides/create_db_resources.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: "mcs"
+page_title: "Create Database Resources"
+description: |-
+  Create Database Resources
+---
+
+# Create Database Instance
+This example shows how to create a simple database instance from scratch.
+First you have to set up basic configuration as shown in [Getting Started](/terraform/docs/providers/mcs/guides/getting_started.html)
```

Comment:
are you sure it's the correct path? why do you think it should start with `/terrafrom/`?

Reply:
Fixed